docs/docs/dev-server/plugins/esbuild.md

```diff
@@ -45,15 +45,16 @@ type Loader =
 
 interface EsbuildPluginArgs {
   target?: string | string[];
+  js?: boolean;
   ts?: boolean;
   json?: boolean;
   jsx?: boolean;
   tsx?: boolean;
   jsxFactory?: string;
   jsxFragment?: string;
   loaders?: Record<string, Loader>;
-  strict?: boolean | Strict[];
   define?: { [key: string]: string };
+  tsconfig?: string;
```

Comment:
I copy and pasted from the source code. I guess `strict` was removed and `js` was added since this doc was updated.